cabify-solution/src/clients/getCredit.js

```diff
@@ -0,0 +1,4 @@
+import Budget from "../models/budget.js";
+
+
+export default (conditions = {}) => Budget.find(conditions)
```

Comment:
This is can potentially return many results, use findOne when you need to find "one correct result" (thats why you had to work with arrays later currentAmount[0]